week03/assignment03_test.go

```diff
@@ -0,0 +1,192 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestRate(t *testing.T) {
+
+	var tests = []struct {
+		a    Movie
+		want error
+	}{
+		{Movie{Name: "Gladiator", Length: 155, plays: 4}, nil},
+		{Movie{Name: "Léon: The Professional", Length: 110}, fmt.Errorf("can't review a movie without watching it first")},
+	}
+
+	for _, tt := range tests {
+		t.Run("", func(t *testing.T) {
+			got := tt.a.Rate(50.2)
+			if got != tt.want {
```

Comment:
As we will learn next week, we want to be careful when checking errors this exactly. Instead, it is considered idiomatic to confirm whether an error is `nil` or not.

Reply:
ok